packages/axios-extension/src/abap/adt/handlers/transport.ts

```diff
@@ -0,0 +1,97 @@
+import XmlParser from 'fast-xml-parser';
+import * as xpath from 'xpath';
+import { DOMParser } from '@xmldom/xmldom';
+
+export function parseTransportRequests(xml: string): string[] {
+    if (XmlParser.validate(xml) !== true) {
+        return [];
```

Comment:
Log something here? That probably means that the logger needs to be passed in somehow

Reply:
Now you see the axios extension doesn't have looger passed in :P

Reply:
`ServiceProvider` has an instance of logger: https://github.com/SAP/open-ux-tools/blob/9d4e3bcfd4a1d45e93ba603ecf345e34896cedf8/packages/axios-extension/src/base/service-provider.ts#L25

Reply:
I added this one. It doesn't look like it instantiate from YUI logger tho.

Reply:
> I added this one. It doesn't look like it instantiate from YUI logger 

Not sure I understand but do we need to make the `logger` module more flexible or what is missing?

Reply:
For generators used in YUI, the vscode logger is passed into the generator as option. The log messages can be found on the output tab in vscode. Regular logs are not accessible on the console.